src/main/java/com/jensjansson/ce/views/persons/EmployeesView.java

```diff
@@ -60,17 +62,31 @@ public void updateGrid(Person person) {
                     public void stopEditingPerson() {
                         dialog.close();
                     }
+
+                    @Override
+                    public void deletePerson() {
+                        dialog.close();
+                        Notification notification = new Notification(
+                                "Delete has been disabled for demo purposes. "
+                                        + "Otherwise we would not have any items left.");
```

Comment:
IMO we shouldn't add a delete button which doesn't do anything. I see that @anezthes has added it in the Figma design, but maybe we can just ignore it?

Reply:
@anezthes also added it to the code. I think we should try to show as realistic an example as possible, and in a CRUD view, you can usually also delete.  I don't have a strong opinion, but I would prefer to go as far as possible to mimic real apps.

Reply:
Having a button that basically just says "This doesn't work because this is a demo, not a real app" is IMO a big step away from realistic app. :D

But if you both think it's better to have it like this, I'm not fighting against it.

Reply:
Based on CRUD's editor. Whether or not you want placeholders in the app is up to you. I think it can serve a purpose. "Where should we put the delete button?" "I know, right click, press up, up, down, down, left, right, left, right, B, A and voilá! It's deleted." :D

I don't think it hurts, but your call.

Reply:
Maybe the biggest reason why I'm against it is that I actually felt some negative feelings when I clicked the button.
First of all I felt some disappointment because the app didn't do what I asked for.
And then this explanation: "Otherwise we would not have any items left."
My first thought to that was: "Well why don't you finish your app by adding also a form for creating new items, instead of making excuses."

> I don't think it hurts, but your call.

If it's 2 against 1, I'm not going to be a dictator and order you to do it the way I want. :) Unless this message made either of you change your mind, let's keep the button.

Reply:
And btw, I do understand the point that it's good to show an example where a delete button could be, and it's a good point. 

But all things considered, it's a trade-off and to me personally the downside is bigger.

Reply:
I toned down the message a bit to be less explaining, which gave negative feelings.